data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,753 @@
+					goto silence
+				`	"Hello! I'm fairly new to this whole 'conquering the galaxy' thing, but I think that beating that crew of scoundrels like I did makes me the new boss of this place, right?"`
+			`	The colonists exchange looks. They seem far too lost in their own misery to even try dealing with your obnoxiously chipper demeanor, and you're simply given a chorus of shrugs and murmured "Yah, sure, why not" responses. "Whatever you say, chief," one mutters.`
+				goto tribute
+			label silence
+			`	Painfully thick silence stretches between you and the welcoming party as everybody tries to figure out what to do, or say in response to your arrival. It's finally broken by a loud bleating.`
+			`	"We brought you a goat," one of the villagers mentions in an attempt to be helpful. He's promptly slapped on the back of the head by another member of the delegation, silencing him quite thoroughly.`
+			label tribute
+			`	A young man at the head of the delegation steps forward and speaks. "Hey, chief - landlord - your majesty." Now that you've had a moment to look them over, it appears that none of the people here are much older than thirty. "We don't have much. That crew you spaced had already taken our crops and sold 'em offworld. All we've really got is..." The goat's fixed with a scornful gaze.`
+			`	"Anyways. We know how this works, you're not the first landlord we've had, and I doubt you'll be the last. We'll scavenge the last gang's camp and gather up any valuables for you, will that cover this month's rents? It'll take a few days, but come back then and we'll have something worthwhile for you. I swear, we really don't have anything else worth taking. We're already down to using hand tools when our ancestors came here with tractors..."`
+			choice
+				`	"You can keep the goat. I'll be back in a few days to collect."`
+					accept
+				`	"That's... pathetic. I expected more. I'm taking your goat, and I expect you sorry bunch of mud-scrapers to have some proper loot by the time I'm back."`
+					accept
+				`	"You know, I don't think this whole 'conquering tyrant' thing is going to work out from me. Can we just forget the whole tribute debacle? I'll leave you all alone. Sorry for the trouble."`
+			`	"... You're messing with us, right?" Only a few of the colonists have the barest hints of hope visible on their faces. "That's... Well, it would be mighty fine to actually be able to keep something for a change. Alright!"`
+			`	You can see the barest clue of a smile on the young man's lips, and light in his eyes that weren't there when you'd arrived. "And hey! You should come back in a couple days anyways, we don't have much but we'd like to throw you some sort of party to show our thanks. We make a mean corn liquor in these parts, and, well... We really hated that last crew that was camped out here. It'd mean a lot to us if you came by."`
+			`(To release the planet from tribute, select it from orbit and use the hail key.)`
```

Comment:
I like to avoid adding "gameplay tips" to conversations wherever possible. If this mission is changed to not use tributing, then we don't need this tip.